cluster/test-smoke.sh

```diff
@@ -23,10 +23,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-TEST_ARGS="$@"
+TEST_ARGS=("$@")
```

Comment:
Why did I do that: 
```
In ./cluster/test-smoke.sh line 28:
TEST_ARGS="$@"
          ^--^ SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```

Reply:
doing this only to expand as `TEST_ARGS[0]` down below seems like needless indirection, I feel like there's got to be a better way to express this?

Reply:
There should be no problem with this modification.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2124

Reply:
```suggestion
TEST_ARGS="$*"
```
Wouldn't that work just as easy?

Reply:
Has been updated.

Reply:
Sorry, I shouldn't have revised it all based on the shellcheck example.

Reply:
/cc cblecker